features/ui/footer/footer.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components";
+import { Routes } from "@config/routes";
+import { color, breakpoint, textFont, space } from "@styles/theme";
+import { version } from "config/version";
+import { FooterItemLink } from "./footer-item-link";
+
+const footerItems = [
+  { text: "Docs", href: Routes.docs },
+  { text: "API", href: Routes.api },
+  { text: "Help", href: Routes.help },
+  { text: "Community", href: Routes.community },
+];
+
+const Container = styled.footer`
+  display: grid;
```

Comment:
An alternative solution would be to use `display: flex` here. It's a bit simpler imo.